.github/workflows/build.yml

```diff
@@ -191,35 +199,9 @@ jobs:
         uses: actions/setup-python@v5
         with:
           python-version: ${{ steps.setup-env.outputs.python-version }}
-      - uses: actions/cache@v3
-        env:
-          BASE_CACHE_KEY: "${{ github.job }}-${{ runner.os }}-\
-            py${{ steps.setup-python.outputs.python-version }}-\
-            packer${{ steps.setup-env.outputs.packer-version }}-"
+      - uses: hashicorp/setup-packer@v3
```

Comment:
You've completely removed the `actions/cache` step here in https://github.com/cisagov/skeleton-packer/commit/8666ad511a34280248950a371135b731aa354b83 and I'm guessing that wasn't intentional.